stories/__tests__/SQFormScrollableCard.stories.test.tsx

```diff
@@ -138,11 +138,9 @@ describe('SQFormScrollableCard', () => {
     });
 
     it('Should not render specified header title when disabled', () => {
-      render(<Default title="Testing Title" isHeaderDisabled={true} />);
+      render(<Default title={undefined} />);
```

Comment:
I think instead of explicitly passing in `undefined` most people will just not pass it in, so can we leave it out or make it empty string like the comment says?

Reply:
Agreed.  I believe that I do need to pass the `title` here in order to override the default value given to the `Default` component, but I agree that passing in an empty string makes more sense.  Will update this.